maintenance_collaboration.Rmd

```diff
@@ -31,24 +31,25 @@ And delete the package current code of conduct if there was one.
 
 You can use issue, pull request and contributing guidelines.
 Having a contributing file as `.github/CONTRIBUTING.md` or `docs/CONTRIBUTING.md` is compulsory.
+An easy way to insert a template for a contributing guide is the [`use_tidy_contributing()` function from the `usethis` package](https://usethis.r-lib.org/reference/tidyverse.html), which inserts [this template](https://github.com/r-lib/usethis/blob/main/inst/templates/tidy-contributing.md) as `.github/CONTRIBUTING.md`.
```

Comment:
Note that it has to be tweaked? E.g. it refers another COC

Reply:
good point, thanks! New sentence added to state that templates will need to be modified for CoC.